apps/service_providers/llm_service/main.py

```diff
@@ -49,6 +63,11 @@ def transcribe_audio(self, audio: BytesIO) -> str:
         )
         return transcript.text
 
+    def get_callback_handler(self, llm_model: BaseLanguageModel) -> BaseCallbackHandler:
+        from langchain_community.callbacks import OpenAICallbackHandler
+
+        return OpenAICallbackHandler()
+
 
 class AzureLlmService(LlmService):
```

Comment:
Can this one use the OpenAICallbackHandler? It's hard to find in the docs...

Reply:
I'm not sure. I don't think their API returns the token usage.